src/app.js

```diff
@@ -17,6 +17,25 @@ const form = document.querySelector(".diff-form");
 const result = document.querySelector(".result");
 const popContainer = document.querySelector(".pop-container");
 
+//adding event listener to initialize Date when page is loaded
+// Todo: Below Event Listener can be replaced with jQuery document.ready() but require jQuery lib
+document.addEventListener('DOMContentLoaded', function() {
+    // Get current date in GMT
+    var d = new Date();
+    var nd = d.toLocaleString('en-US', { timeZone: 'America/New_York' });
```

Comment:
Is this nd required? I think it can be removed safely.

Reply:
Sorry, my bad! Line 25 is unrequired as the variable "nd" is never used (I created it while testing but forgot to remove it).
```suggestion
```